CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_hubspot v0.16.0
+
+## 🎉 Feature Update 🎉 
+- This release supports running the package on multiple Hubspot sources at once! See the [README](https://github.com/fivetran/dbt_hubspot?tab=readme-ov-file#step-3-define-database-and-schema-variables) for details on how to leverage this feature ([PR #133](https://github.com/fivetran/dbt_hubspot/pull/133)).
```

Comment:
I think it is also very relevant to include here that customers will not also see a new `source_relation` field in their end models.

Additionally, we should flag that customers using the ticket models will need to run a full refresh to capture the new schema change.

Reply:
added